guides/common/modules/ref_system-requirements.adoc

```diff
@@ -4,11 +4,19 @@
 The following requirements apply to the networked base operating system:
 
 * x86_64 architecture
-ifeval::["{build}" == "satellite"]
+ifdef::satellite[]
 * The latest version of Red Hat Enterprise Linux 7 Server
 endif::[]
 * 4-core 2.0 GHz CPU at a minimum
 
+ifdef::foreman-el,foreman-deb[]
+ifeval::["{context}" == "{project-context}"]
+* A minimum of 4 GB RAM is required for {ProjectServer} to function.
+{Project} running with less RAM than the minimum value might not operate correctly.
+endif::[]
```

Comment:
I don't understand why you used `{context}` and not simply `context`:
```
doc-Installing_Server_on_Debian/master.adoc
9::context: {project-context}

doc-Installing_Server_on_Red_Hat/master.adoc
11::context: {project-context}
```

Reply:
I don't either. I just copied this from somewhere :)